x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -129,6 +128,11 @@ export function ProcessTreeNode({
     : 'sessionView:processTreeNodeForkIcon';
   const processIcon = hasExec ? 'console' : 'branch';
 
+  const timeStampsNormal = formatDate(
+    process.getDetails().process.start,
+    'MMM DD, YYYY @ hh:mm:ss.SSS'
```

Comment:
probably good to store this format somewhere more common to be shared with other components (e.g. in `common/constants.ts`